src/xenia/gpu/xenos.h

```diff
@@ -157,8 +157,10 @@ enum class ColorRenderTargetFormat : uint32_t {
 };
 
 enum class DepthRenderTargetFormat : uint32_t {
-  kD24S8 = 0,
-  kD24FS8 = 1,
+  //  kD24S8 = 0,
+  //  kD24FS8 = 1,
+  kD32S8 = 0,
```

Comment:
Don't change these names! They represent the meanings of these values to the Xenos chip, not our underlying implementations!